tests/compartmentReport.cpp

```diff
@@ -215,6 +215,8 @@ void test_compare( const brion::URI& uri1, const brion::URI& uri2 )
 
         BOOST_CHECK( frame1 );
         BOOST_CHECK( frame2 );
```

Comment:
You can also use BOOST_REQUIRE is you don't want to continue with the test, which is probably the case here.